Assignment/palindrome.c

```diff
@@ -8,22 +8,28 @@
 
 #include "palindrome.h"
 #include <ctype.h>
+#include <string.h>
 
 int PalindromeCheck (char str[]) {

```

Comment:
This function returns PALINDROME if I give an empty string or string full of special characters as input.